docs/src/debugging.rst

```diff
@@ -1,8 +1,11 @@
-********************************
-Debugging (Django Debug Toolbar)
-********************************
+*********
+Debugging
+*********
 
-We use the :doc:`django-debug-toolbar:index` to debug the CMS.
+Django Debug Toolbar
+====================
+
+We use the :doc:`django-debug-toolbar:index` to debug the CMS in the browser.
 When :setting:`django:DEBUG` is ``True``, on every HTML page a sidebar is rendered which contains
```

Comment:
> When `DEBUG` is `True`,

Would it make sense to make it more obvious that this is a setting? It is linked in the rendered HTML, but there is no visual distinction and I was confused for a moment. I think for someone completely new this might benefit from being clearer what `DEBUG` actually alludes to and where/how it can be set

Reply:
Added a hint.